packages/dashmate/src/listr/tasks/setup/regular/verifySystemRequirementsTaskFactory.js

```diff
@@ -0,0 +1,123 @@
+import chalk from 'chalk';
+import { Listr } from 'listr2';
+import os from 'os';
+import * as diskusage from 'diskusage';
+
+/**
+ *
+ * @param {Docker} docker
+ * @param {DockerCompose} dockerCompose
+ * @return {verifySystemRequirementsTask}
+ */
+export default function verifySystemRequirementsTaskFactory(docker, dockerCompose) {
+  /**
+   * @typedef {function} verifySystemRequirementsTask
+   * @returns {Listr}
+   */
+  async function verifySystemRequirementsTask() {
+    return new Listr([
+      {
+        title: 'System requirements',
+        task: async (ctx, task) => {
+          await dockerCompose.throwErrorIfNotInstalled();
+
+          const MINIMUM_CPU_CORES = ctx.isHP ? 4 : 2;
```

Comment:
The minimal for MN is 2 CPUs, 60 GB of disk space and 4GB ram + 2GB of swap possibly

I guess for full node should be less